### PR TITLE
ABOS_SOFS lftp job

### PR DESCRIPTION
**Pull request by danfruehauf**
Downloads files to WIP, then pushes the new ones to incoming
directory.

### CODE REVIEW

File: ABOS/lftp_sync.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+declare -r FTP_PASSWORD=$IMOS_PO_CREDS_BOM_FTP_PASSWORD
+declare -r YEAR=`date +%Y`
+declare -r FTP_DIR="/register/bom404/outgoing/IMOS/MOORINGS/$YEAR"
+FTP_EXTRA_OPTS=""
+
+declare -r ABOS_SOFS_WIP_DIR=$WIP_DIR/ABOS/SOFS
+declare -r ABOS_SOFS_WIP_DIR_FTP=$ABOS_SOFS_WIP_DIR/ftp
+declare -r ABOS_SOFS_INCOMING_DIR=$INCOMING_DIR/ABOS/SOFS
+
+# sync files from remote AATAMS ftp server
```
Comment (mhidas) [inline ABOS/lftp_sync.sh]: BoM, not AATAMS

File: ABOS/lftp_sync.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# $1 - lftp log file
+sync_files() {
+    local lftp_log_file=$1; shift
+    lftp -e "open -u $FTP_USER,$FTP_PASSWORD $FTP_SOURCE; mirror -e --parallel=10 $FTP_EXTRA_OPTS --exclude-glob *_ODV.zip -vvv --log=$lftp_log_file $FTP_DIR $ABOS_SOFS_WIP_DIR_FTP; quit"
```
Comment (mhidas) [inline ABOS/lftp_sync.sh]: I don't think we need `--exclude-glob *_ODV.zip`. Instead, we should include only netCDF files and directories, i.e. `--include-glob='*.nc' --include-glob='*/'`

**Comment by mhidas** [comment]
@danfruehauf I'll just make the changes.

File: ABOS/lftp_sync.sh (hunk 2 above)
Comment (mhidas) [inline ABOS/lftp_sync.sh]: Actually we don't need any include/exclude, as we explicitly only push netcdf files to incoming.

**Comment by mhidas** [comment]
@danfruehauf See what you think. Then we can squash & merge.

**Comment by mhidas** [comment]
Sorry, had a couple of typos in there

**Comment by danfruehauf** [comment]
@mhidas Ah! You fixed it all. Yeah, lets squash and merge it. It was done mostly to save you time...

**Comment by mhidas** [comment]
@danfruehauf GTG

**Comment by danfruehauf** [comment]
Merging this. The cronjob is disabled anyway :)

**Pull request merged**